src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -31,29 +31,34 @@
 
 
 DATA_SAVE_ID  = wx.NewId()
-
+# NOTE: for reference, only comment out unwanted icons
+CUSTOMIZED_TOOLITEMS = (
+        ('Home', 'Reset original view', 'home', 'home'),
+        ('Back', 'Back to  previous view', 'back', 'back'),
+        ('Forward', 'Forward to next view', 'forward', 'forward'),
+        (None, None, None, None),
+        ('Pan', 'Pan axes with left mouse, zoom with right', 'move', 'pan'),
+        ('Zoom', 'Zoom to rectangle', 'zoom_to_rect', 'zoom'),
+        #('Subplots', 'Configure subplots', 'subplots', 'configure_subplots'),
+        (None, None, None, None),
+        #('Save', 'Save the figure', 'filesave', 'save_figure')
+        )
```

Comment:
It is better to filter the standard matplotlib toolitems instead of copying them.  That way the code would be OK and up to date should matplotlib add some icon or rename handler method.  See the next comment.